fluent-react/src/markup.js

```diff
@@ -1,6 +1,12 @@
-/* eslint-env browser */
-
-const TEMPLATE = document.createElement("template");
+/* global window, require */
+let doc = null;
+if (typeof(window) === "undefined" || !window || !window.document) {
+  const jsdom = require("jsdom");
+  doc = new jsdom.JSDOM("<html><body></body></html>").window.document;
```

Comment:
How about we use the minimal valid HTML5 document here?

    <!doctype html><title>JSDOM</title>